init-odkx-sync-endpoint.py

```diff
@@ -18,71 +19,61 @@ def run_interactive_config():
 
     try:
         domain, email = parse_env_file(env_file_location)
-        print("Found configuration at {}".format(env_file_location))
+        typer.echo(f"Found configuration at {env_file_location}")
     except OSError:
-        print("No default https configuration file found at expected path {}. This prevents automatically renewing certs!".format(env_file_location))
-        print("Please check your paths and file permissions, and make sure your config repo is up to date.")
-        exit(1)
+        typer.echo(f"No default https configuration file found at expected path {env_file_location}. This prevents automatically renewing certs!")
+        typer.echo("Please check your paths and file permissions, and make sure your config repo is up to date.")
+        raise typer.Exit()
 
-    print("Welcome to the ODK-X sync endpoint installation!")
-    print("This script will guide you through setting up your installation")
-    print("We'll need some information from you to get started though...")
+    typer.echo("Welcome to the ODK-X sync endpoint installation!")
+    typer.echo("This script will guide you through setting up your installation")
+    typer.echo("We'll need some information from you to get started though...")
     time.sleep(1)
-    print("")
-    print("Please input the domain name you will use for this installation. A valid domain name is required for HTTPS without distributing custom certificates.")
-    input_domain = input("domain [({})]:".format(domain))
+    typer.echo("")
+    typer.echo("Please input the domain name you will use for this installation. A valid domain name is required for HTTPS without distributing custom certificates.")
+    input_domain = typer.prompt(f"domain [({domain})]")
 
     if input_domain != "":
         domain = input_domain
 
-    print("")
-    use_custom_password = input("Do you want to use a custom LDAP administration password (y/N)?")
-    if use_custom_password == "y":
-        print("")
-        print("Please input the password to use for ldap admin")
-        default_ldap_pwd = input("Ldap admin password:")
+    typer.echo("")
+    use_custom_password = typer.confirm("Do you want to use a custom LDAP administration password?")
```

Comment:
It looks like the underlying click call allows setting if the default is true or false, so I think there should be a way to get it to work via typer, even if we have to get the underlying click object and then modify it directly. https://click.palletsprojects.com/en/8.1.x/api/#click.confirm

Reply:
I just tested this, and just setting the `default=` parameter will do the right thing, and get the value passed down to click via typer.

Reply:
thank you @matt9j 
u sing `click`'s  `default` functionality makes life easier